### PR TITLE
LibreCAD: fix missing built-in plugins menu

### PR DESCRIPTION
**Pull request by hervyqa**
Solved #32497

#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)

https://user-images.githubusercontent.com/45872139/147313649-5c5f7710-8f24-4e88-8f6b-b0d7e60a6aee.mp4

### CODE REVIEW

**Comment by hervyqa** [comment]
@abenson. is there any improv?

File: srcpkgs/LibreCAD/template [hunk 1]
```diff
@@ -46,9 +46,14 @@ do_install() {
...
 		vcopy unix/resources/${resource} usr/share/librecad/
 	done
 
-    vcopy unix/resources/plugins usr/lib/librecad/
+	# install plugins
+	vmkdir usr/share/librecad/plugins
+	vcopy unix/resources/plugins usr/lib/librecad/
+	for plugins in libalign libasciifile libimportshp liblist libpicfile libplotequation libsameprop libsample; do
+		ln -sf ../../../lib/librecad/${plugins}.so ${DESTDIR}/usr/share/librecad/plugins/${plugins}.so
+	done
```
Comment (ericonr) [inline srcpkgs/LibreCAD/template:58]:
```suggestion
	for plugin in $DESTDIR/usr/lib/librecad/*.so ; do
		ln -sf ../../../lib/librecad/$(basename $plugin) ${DESTDIR}/usr/share/librecad/plugins/$(basename $plugin)
	done
```

Something like this should work, what do you think?

Comment (hervyqa) [reply srcpkgs/LibreCAD/template:58]: I agree, looks simpler.

Comment (hervyqa) [reply srcpkgs/LibreCAD/template]: Then how to rebase to single commit?

**Comment by ericonr** [comment]
Just `git commit --amend` is enough for most cases, to avoid making a new commit at all.

What commands did you run for this to happen?

`git rebase upstream/master -i; git push -f` should be enough to fix, if you squash the commits.

**Comment by hervyqa** [comment]
> Just `git commit --amend` is enough for most cases, to avoid making a new commit at all.
> 
> What commands did you run for this to happen?
> 
> `git rebase upstream/master -i; git push -f` should be enough to fix, if you squash the commits.

that is my mistake. first I clicked the fetch button from github, then git pull to the LibreCAD branch. It turns out that all commits from master are joined to the LibreCAD branch. Then I tried to push, it didn't come back to https://github.com/void-linux/void-packages/commit/6daf0ff.
sorry @ericonr, I'm still a beginner with git. any suggestions to go back to https://github.com/void-linux/void-packages/commit/6daf0ff and skip commits afterwards? because of this I feel guilty.

**Comment by hervyqa** [comment]
Finally, it's fixed with a single commit :)

**Comment by ericonr** [comment]
The GitHub interface can be confusing for some stuff, no worries! It's an easy mistake to make.

**Comment by ericonr** [comment]
Cheers, it's working great :)

**Pull request merged**